apps/cli/src/node/docker-compose-dev.yaml

```
@@ -160,11 +160,30 @@ services:
         depends_on:
             validator:
                 condition: service_healthy
+        healthcheck:
+            test: ["CMD", "traefik", "healthcheck"]
+            interval: 5s
+            timeout: 1s
+            retries: 5
         ports:
             - 8080:8080
-            - 8088:8088
         volumes:
             - ${SUNODO_BIN_PATH}/node/traefik:/etc/traefik
 
+    prompt:
+        image: debian:bookworm-20230814
```

the user should already have `debian:bookworm-slim` because of `cartesi/rollups-node`.

https://github.com/cartesi/rollups-node/blob/main/build/Dockerfile#L18C7-L18C26

```suggestion
        image: debian:bookworm-slim
```